calico/networking/vxlan-ipip.md

```diff
@@ -44,6 +44,7 @@ Encapsulation of workload traffic is typically required only when traffic crosse
 **Limitations** 
 
 - IP in IP supports only IPv4 addresses
+- VXLAN in IPv6 supports only for kernel >= 3.12
```

Comment:
nit: `VXLAN in IPv6 is only supported for kernel versions >= 3.12`

Reply:
sorry😄 I'm not a english native speaker. Now updated.